plugins/jira/jira.plugin.zsh

```diff
@@ -38,8 +42,11 @@ open_jira_issue () {
   fi
 
   if [ -z "$1" ]; then
-    echo "Opening new issue"
-    $open_cmd "${jira_url}/secure/CreateIssue!default.jspa"
+    if [[ "x$JIRA_DASHBOARD" = "xtrue" ]]; then
```

Comment:
You don't need the "x$FOO" trick in `zsh`. `[[ "$JIRA_DASHBOARD == true ]]` is sufficient.